R/zchunk_LA124.heat.R

```diff
@@ -40,70 +40,175 @@ module_energy_LA124.heat_DISABLED <- function(command, ...) {
     L1231.out_EJ_R_elec_F_tech_Yh <- get_data(all_data, "temp-data-inject/L1231.out_EJ_R_elec_F_tech_Yh")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+    # Create list of heat regions
+    heat_regionIDs <- A_regions$GCAM_region_ID[ A_regions$heat == 1]
```

Comment:
This is _not_ selecting the USA region.  It is selecting all those regions for which `A_regions$heat` is 1.  This could be clarified a bit by filtering the table based on the `heat` column and then extracting the region column.